gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```

Comment:
I assume updating to the latest gradle 8.1.1 is not possible here?

Reply:
I didn't even try it. I assumed it wouldn't work in 22.7 but it seems to work just fine.